code/modules/mob/living/simple_animal/abnormality/he/siren.dm

```diff
@@ -0,0 +1,153 @@
+//Code by Coxswain sprites by Sky
+/mob/living/simple_animal/hostile/abnormality/siren
+	name = "Siren"
+	desc = "The siren that sings the past."
+	icon = 'ModularTegustation/Teguicons/64x64.dmi'
+	pixel_x = -16
+	base_pixel_x = -16
+	icon_state = "siren"
+	maxHealth = 1000
+	health = 1000
+	threat_level = HE_LEVEL
+	start_qliphoth = 5
+	minimum_distance = 3 //runs away during pink midnight
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = 15, //you can't spam insight so 0 is a bit much
+		ABNORMALITY_WORK_INSIGHT = 80,
+		ABNORMALITY_WORK_ATTACHMENT = 40,
+		ABNORMALITY_WORK_REPRESSION = 50
+		)
+	work_damage_amount = 11
+	work_damage_type = WHITE_DAMAGE
+
+	ego_list = list(
+		/datum/ego_datum/weapon/song,
+		/datum/ego_datum/weapon/songmini,
+		/datum/ego_datum/armor/song
+		)
+	gift_type = /datum/ego_gifts/song
+
+//meltdown effects
+	var/meltdown_cooldown_time = 144 SECONDS
+	var/meltdown_cooldown
+	var/song_cooldown_time = 60 SECONDS
+	var/song_cooldown
+	var/meltdown_imminent = FALSE
+	pet_bonus = "beeps" //saves a few lines of code by allowing funpet() to be called by attack_hand()
+//Post-work effect
+	var/work_effect
+	var/list/unsafe = list()
+//SFX
+	var/datum/looping_sound/siren_musictime/musictime
+	var/playstatus = 0
+	var/playrange = 40
+
+//Insight work causes the user to have every buff and debuff removed
+/mob/living/simple_animal/hostile/abnormality/siren/AttemptWork(mob/living/carbon/human/user, work_type)
+	if(work_type == ABNORMALITY_WORK_INSIGHT)
+		work_effect = TRUE
+		if(!playstatus)
+			playsound(loc, 'sound/abnormalities/siren/backtherebenjamin.ogg', 50, FALSE)
+			playstatus = 1 //prevents song overlap
```

Comment:
You might want to set it to defines instead of 1 and 0, so, TRUE and FALSE respectively.